src/api/typePolicies/typePolicies.base.ts

```diff
@@ -1,3 +1,35 @@
 import { TypePolicies } from '@apollo/client';
+import { ProjectListOutput, QueryProjectsArgs } from '..';
 
-export const typePolicies: TypePolicies = {};
+export const typePolicies: TypePolicies = {
+  Query: {
+    fields: {
+      projects: {
+        merge(
+          existing: ProjectListOutput | undefined,
+          incoming: ProjectListOutput,
+          { args }: { args: QueryProjectsArgs | null }
+        ) {
+          const count = args?.input?.count;
+          const page = args?.input?.page;
+          const DEFAULT_COUNT = 10;
+          const existingProjects = existing?.items ?? [];
+          const startNew = (page ?? 1) * (count ?? DEFAULT_COUNT) + 1;
+          const endNew = (page ?? 1) * (count ?? DEFAULT_COUNT);
+          const updatedProjects = [
+            ...existingProjects.slice(0, startNew),
+            ...incoming.items,
+            ...existingProjects.slice(endNew),
+          ];
+          return {
+            ...incoming,
+            items: updatedProjects,
+          };
+        },
+        read(existing: ProjectListOutput | undefined) {
+          return existing;
+        },
```

Comment:
What's this read do?

Reply:
I would have thought nothing, so at first I left it out. It then returned no data. Seems almost like a bug

Reply:
Hm, I just tried it again, and it worked fine without it. Maybe a fluke before?